exotica_core/include/exotica_core/problems/dynamic_time_indexed_shooting_problem.h

```diff
@@ -33,6 +33,8 @@
 #include <exotica_core/dynamics_solver.h>
 #include <exotica_core/planning_problem.h>
 #include <exotica_core/tasks.h>
+#include <boost/random.hpp>
```

Comment:
Don't use boost. It's in std: [`<random>`](http://www.cplusplus.com/reference/random/normal_distribution/)